src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java

```diff
@@ -308,17 +314,17 @@ public void uncheckedReadSkipBackOne() {
 
     @NotNull
     @Override
-    public Bytes<Underlying> writeSkip(long bytesToSkip)
+    public Bytes<U> writeSkip(long bytesToSkip)
             throws BufferOverflowException, IllegalStateException {
-        final long writePosition = writePosition();
-        writeCheckOffset(writePosition, bytesToSkip);
-        uncheckedWritePosition(writePosition + bytesToSkip);
+        final long writePos = writePosition();
+        writeCheckOffset(writePos, bytesToSkip);
+        uncheckedWritePosition(writePos + bytesToSkip);
```

Comment:
I like longer variable names, this isn't a good example, but I think abbreviations unnecessarily hurt search-ability